go get fails

Running `go get -u github.com/Azure/azure-service-bus-go` currently fails. To reproduce:

```
~ docker run -it --rm golang:1.12
root@8cf7ee22fa2e:/go# go get -u github.com/Azure/azure-service-bus-go
# github.com/Azure/azure-service-bus-go
src/github.com/Azure/azure-service-bus-go/rpc.go:131:13: undefined: rpc.LinkOption
src/github.com/Azure/azure-service-bus-go/rpc.go:133:23: undefined: rpc.LinkWithSessionFilter
src/github.com/Azure/azure-service-bus-go/rpc.go:137:26: too many arguments in call to rpc.NewLink
src/github.com/Azure/azure-service-bus-go/rpc.go:420:13: undefined: rpc.LinkOption
src/github.com/Azure/azure-service-bus-go/rpc.go:436:23: undefined: rpc.LinkWithSessionFilter
src/github.com/Azure/azure-service-bus-go/rpc.go:446:26: too many arguments in call to rpc.NewLink
```

--- a/changelog.md
+++ b/changelog.md
@@ -2,6 +2,9 @@
 
 ## `head`
 
+## `v0.9.1`
+- bump common version to v2.1.0
+
 ## `v0.9.0`
 - periodically refresh claims based auth for connections to resolve [issue #116](https://github.com/Azure/azure-service-bus-go/issues/116)
 - refactor management functionality for entities into composition structs

--- a/go.mod
+++ b/go.mod
@@ -3,7 +3,7 @@ module github.com/Azure/azure-service-bus-go
 go 1.12
 
 require (
-	github.com/Azure/azure-amqp-common-go/v2 v2.0.1-0.20190619212702-9727f6e7bc50
+	github.com/Azure/azure-amqp-common-go/v2 v2.1.0
 	github.com/Azure/azure-sdk-for-go v30.1.0+incompatible
 	github.com/Azure/go-autorest v12.0.0+incompatible
 	github.com/devigned/tab v0.1.1

--- a/iterator_test.go
+++ b/iterator_test.go
@@ -41,7 +41,7 @@ func (suite *serviceBusSuite) TestMessageIterator() {
 
 			q, err := ns.NewQueue(queueName)
 			defer func() {
-				q.Close(context.Background())
+				_ = q.Close(context.Background())
 			}()
 			suite.NoError(err)
 
@@ -197,6 +197,7 @@ func testMessageIteratorLargePageSize(ctx context.Context, t *testing.T, queue *
 	}
 
 	require.NoError(t, queue.Send(ctx, NewMessageFromString(newPayload(pageSize+1))))
+	time.Sleep(1 * time.Second)
 
 	got := uint32(0)
 	for {

--- a/namespace.go
+++ b/namespace.go
@@ -48,7 +48,7 @@ const (
 	//`
 
 	// Version is the semantic version number
-	Version = "0.9.0"
+	Version = "0.9.1"
 
 	rootUserAgent = "/golang-service-bus"
 )

